ext/kv/interface.rs

```diff
@@ -237,6 +237,7 @@ pub struct KvCheck {
 pub struct KvMutation {
   pub key: Vec<u8>,
   pub kind: MutationKind,
+  pub expiration_ms: i64,
```

Comment:
Is this an absolute expiration timestamp? or a delta TTL?

I think we should go with delta TTL, similar to `delay_ms` for queues (at least until we have `Temporal.ZonedDateTime` so that we can deal with zoned date time)

Reply:
I agree.